notebooks/community/ml_ops/stage7/get_started_with_model_monitoring_xgboost.ipynb

```diff
@@ -0,0 +1,1477 @@
+{
```

Comment:
<p>Ah we still don't have a way to check the status of a monitoring job!? :(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>not reliable. What you need to know is when the sampled data on GCS has been moved to BQ table. Hence why the later check for new entries in BQ table.</p>

Reply:
Got it!